content/rs/administering/database-operations/importing-data.md

```diff
@@ -107,6 +107,11 @@ To specify to import from a local mount point on a node:
 
     For example: `/mnt/Public/backup.rbd`
 
+In order to use this option, the storage needs to be mounted in the same path on all cluster nodes.
+The import source folder needs to be owned by the redislabs user with write permissions.
```

Comment:
In line 91 we mention we need read from local and destination, and they should be owned by `redislabs:redislabs`.
In line 111 we mention we need source folder to be owned by redislabs user and have write permissions.

1. Are `source folder` and `local mount point` not the same folder? even if `source folder` refers to the folder and `local mount point` refers to the RDB file itself, they should be named similarly.
2. We should probably put all the required permissions in one paragraph, for reader comfort.

Also, I suggest that @yaeltzirulnikov be queried on this doc, since she is more familiar with the feature than me.